src/components/views/rooms/RoomTile.js

```diff
@@ -1,6 +1,7 @@
 /*
 Copyright 2015, 2016 OpenMarket Ltd
 Copyright 2017 New Vector Ltd
+Copyright 2018 Michael Telatynski <7t3chguy@gmail.com>
```

Comment:
Are you you or are you New Vector atm?

Reply:
This wasn't a `p1` issue and thus not chargeable hours so `<Me/>`

Reply:
👍